fatiando/gravmag/transform.py

```diff
@@ -118,6 +125,106 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def mstde(x, y, data, shape):
+    """
+    Multi-Scale Tilt Depth Estimation
+
+    Implements the method of van Buren (2013).
+
+    An empirical method for attempting to approximate the depth of the source
+    of a magnetic anomaly, based on altering tilt and continuation of the
+    magnetic field intensity.
+
+    This method will also work well on Total Gradient Amplitude, but care must
+    be taken to reduce noise.
+
+    .. warning::
+
+        This algorithm expects Reduced to Pole data. Within `fatiando`, the
+        recommended method is to use equivalent layers to achieve this. See
+        `cookbook/gravmag_eqlayer_pel_polereduc.py` and
+        `cookbook/gravmag_eqlayer_polereduc.py` for an idea of how this can be
+        done.
+
+    **References**
+
+    Van Buren, Reece. 2013."Multi-Scale Tilt Depth Estimation." MSc Thesis,
+    Johannesberg: University of the Witwatersrand.
+    URI: http://mobile.wiredspace.wits.ac.za/bitstream/handle/10539/14011/
+    MSc_Dissertation_RvanBuren_2013.pdf.
+    """
+    RTP_TDR = tilt(x, y, data, shape)
+    AREA = numpy.arctan2( numpy.tan( (numpy.pi/2)-numpy.arctan2(RTP_TDR) ) )
+    return AREA
+
+
+def tilt(x, y, data, shape):
+    """
+    Calculates the magnetic tilt, as defined by Miller and Singh (1994):
+
+    .. math::
+
+    tilt(f) = tan^{-1}(\\frac{\\frac{dT}{dz}}{\\sqrt{\\frac{dT}{dx}^2 +
+    \\frac{dT}{dy}^2}})
+
+    Parameters:
+
+    * x, y : 1D-arrays
+        The x and y coordinates of the grid points
+    * data : 1D-array
+        The potential field at the grid points
+    * shape : tuple = (ny, nx)
+        The shape of the grid
+
+    Returns:
+
+    * tilt : 1D-array
+        The tilt angle of the total field.
+
+    **References**
+
+    Miller, Hugh G, and Vijay Singh. 1994. "Potential Field Tilt --- a New
+    Concept for Location of Potential Field Sources."
+    Journal of Applied Geophysics 32 (2--3): 213–17.
```

Comment:
@mtb-za keep an eye out for non-ASCII characters in your code because Python can be a bit annoying about it. For example, the dash in "213-17" is not really an ASCII "-". Notice that it's slightly longer. This happens all the time when copy/pasting a reference.